scripts/convert_csv_to_fasta.py

```diff
@@ -0,0 +1,84 @@
+# Script to convert csv files to fasta files
+#
+import csv
+
+def separate_seq(sequence:str) -> str:
+    """ Separate sequence in 60-characters chunks
+         and append new line to each chunk except remaing one
+
+    Args:
+        sequence : protein sequence from csv file
+
+    Returns:
+        fs: separated sequence for fasta file
+    """
+    # Initialize new sequence
+    fs = ""
+    # Count number of full chunks
+    chunk = 60
```

Comment:
I actually saw a source that said all lines should be shorter than 80 characters, up to you if you want to change this to 79. If it works it doesn't matter I guess.

Reply:
The demo file in prose has 60 characters, in biopython 70. I was watching a YouTube video where the guy was in a notebook inserting a new line character every 60 characters. So, I decided to go with 60. I tested the script on sequences smaller than 60 and multiples of 60, like 120. All worked well.

Reply:
Sounds good, we'll stick with 60 from now on then!